client/my-sites/plans-features-main/index.jsx

```diff
@@ -142,8 +144,13 @@ class PlansFeaturesMain extends Component {
 		);
 	}
 
-	getFAQ( site ) {
-		const { translate } = this.props;
+	getFAQ() {
+		const { site, translate } = this.props;
+
+		if ( this.isJetpackSite( site ) ) {
```

Comment:
Has a Jetpack site gotten into this method? It looks like we already check for this case below.